docs/index.md

```diff
@@ -76,6 +76,287 @@ yamlfix will do the following changes in your code:
+
+```toml
+# pyproject.toml
+
+[tool.yamlfix]
+allow_duplicate_keys = true
+line_length = 80
+none_representation = "null"
+```
+
+## Provide config-files
+
+When running `yamlfix` as a standalone cli application it might be desireable to provide a config file containing just the configuration related to `yamlfix`. A cli-argument `-c` (`--config-file`) can be provided multiple times to read configuration values from `toml` formatted files. The rightmost value-files override the value-files preceding them, only trumped by environment variables. No section headers are necessary for these configuration files, as the expected behaviour is, that those files contain only configuration related to `yamlfix`. For example:
+
+```bash
+# run yamlfix with two config files
+yamlfix -c base.toml --config-file environment.toml file.yaml
+```
+
+```toml
+# base.toml
+allow_duplicate_keys = false
+line_length = 100
+```
+
+```toml
+# environment.toml
+allow_duplicate_keys = true
+```
+
+These provided configuration files would result in a merged runtime-configuration of:
+```toml
+# merged configuration
+allow_duplicate_keys = true
+line_length = 100
+```
+
+## Configure environment prefix
+
+Per default `yamlfix`, when run through cli, will read any environment variable that starts with `YAMLFIX_` and apply it to the merged runtime-configuration object. This default value can be overridden with the cli-parameter `--env-prefix`. For example:
+
+```bash
+# set a configuration value with the default prefix
+export YAMLFIX_LINE_LENGTH="300"
+
+# set a configuration value with the custom prefix
+export MY_PREFIX_NONE_REPRESENTATION="~"
+
+# run yamlfix with a custom environment prefix
+yamlfix --env-prefix "MY_PREFIX" file.yaml
+```
+
+These provided arguments and environment variables would result in a merged runtime-configuration of:
+```toml
+# merged configuration
+# default value for line_length stays at: 80
+none_representation = "~"
+```
+
+## Configuration Options
+
+All fields configured in the [YamlfixConfig class](./reference/#yamlfix.model.YamlfixConfig) can be provided through the means mentioned in [Configuration](#configuration). Here are the currently available configuration options with short examples on their impact to provided `yaml`-files.
+
+### Allow Duplicate Keys
+
+Default: `allow_duplicate_keys: bool = True`<br>
+Environment variable override:
+```bash
+export YAMLFIX_ALLOW_DUPLICATE_KEYS="true"
+```
+
+This option enables the [ruyaml duplicate keys check](https://ruyaml.readthedocs.io/en/latest/api.html#duplicate-keys). With this check enabled `yamlfix` will throw an error if the same key is defined more than once in a dict. Before enabling this option, you should be aware, that multiple yaml-anchor merge keys are considered duplicate keys and should be provided as a list instead: https://github.com/pycontribs/ruyaml/issues/43
```

Comment:
This paragraph made me confused. If I set `allow_duplicate_keys` I expect to be able to have duplicate keys in the yaml, but from what I read here, if I set it to `True` it will enable the duplicate check that will raise an exception if there are duplicate keys. 

If I misunderstood the paragraph then I'll ask you to reformulate it to prevent other people of getting the same idea.

If I understood it right then I think that the exception should be raised if `allow_duplicate_keys == False` instead. Keeping the default to `True`

Reply:
Hi @lyz-code I think I got confused while writing that (double negatives), I'll re-phrase it.